release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2127,6 +2127,28 @@ This behavior has been fixed, and the references are now accepted and correctly
 [id="ocp-4-14-olm-bug-fixes"]
 ==== Operator Lifecycle Manager (OLM)
 
+* Before {product-title} 4.14.2, Operator Lifecycle Manager (OLM) could overwrite privileged cluster role bindings used by {product-title} and Kubernetes resources if you created an Operator group with the same name. In {product-title} 4.14.2 and later, if you create an Operator group that conflicts with a privileged cluster role binding, OLM renames the cluster role binding and adds a hash value avoid conflicts. This renaming only occurs if you Operator group conflicts with a privileged cluster role binding. You must ensure that the name of your Operator group does not conflict with other cluster role bindings that exist on the cluster.
```

Comment:
OLM overwrites the `cluster role`, not `cluster role binding`.

Reply:
Thank you for the feedback. I applied your suggested change.